src/Symfony/Component/Lock/Tests/Store/CombinedStoreTest.php

```diff
@@ -41,12 +42,9 @@ protected function getClockDelay()
      */
     public function getStore(): PersistingStoreInterface
     {
-        $redis = new \Predis\Client(array_combine(['host', 'port'], explode(':', getenv('REDIS_HOST')) + [1 => null]));
-        try {
-            $redis->connect();
-        } catch (\Exception $e) {
-            self::markTestSkipped($e->getMessage());
```

Comment:
why removing this skipping ?

Reply:
the test depends on the connection to work so i marked it as an integration test. Since this is an integration test now it makes sense that the test breaks if it cannot establish a connection to redis

Reply:
when running tests locally, this should still be skipped, so I would indeed keep the skip

Reply:
reverted